### ISSUE TITLE
Avoid empty pages in a volume

### ISSUE THREAD

**Issue by bertfrees** [snaekobbi/pipeline-mod-dedicon#57]
I'm not sure whether Arjan is only referring to the "[Blank braille page at the start of the content](https://github.com/snaekobbi/pipeline-mod-dedicon/issues/43)" issue, or whether there are other issues.

Note that there have been issues with empty pages in the past: see https://github.com/snaekobbi/pipeline-mod-dedicon/issues/19.

**Comment by dkager** [comment]
The test book I have here (I'd say pipeline-mod-braille 1.9.14) also has a blank page after the generated TOC.

**Comment by dkager** [comment]
Also a blank page at the end of one particular chapter. The chapter fits exactly on the previous page, but presumably still triggers a page transition even though the fact that the previous page was full also triggered one.

**Comment by bertfrees** [comment]
> blank page after the generated TOC

I think that is what #43 is about right?

**Comment by dkager** [comment]
Yes, though I'm also seeing some on the boundaries of *matter elements. Will have to test this again in 2017 with updated pipeline-mod-braille. I.e. in the follow-up project.

**Comment by PaulRambags** [comment]
With the new pipeline-mod-braille version 1.10.1-SNAPSHOT, unit test `boilerplate-test-1-enabled` produces a certain `<page/>` no longer.
This is a fragment of the second section, I commented out the page that is no longer generated:
```
    <section>
        <!-- <page>  -->
        <page>
            <row>⠨⠓⠕⠕⠋⠙⠎⠞⠥⠅⠀⠼⠁</row>
            <row/>
        </page>
        <page>
            ...
        </page>
    </section>

```
